odbc/src/api/functions.rs

```diff
@@ -663,77 +602,6 @@ pub unsafe extern "C" fn SQLColumnPrivilegesW(
     unimpl!(statement_handle);
 }
 
-///
-/// [`SQLColumns`]: https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/SQLColumns-function
-///
-/// # Safety
-/// Because this is a C-interface, this is necessarily unsafe
-///
-#[named]
-#[no_mangle]
-pub unsafe extern "C" fn SQLColumns(
```

Comment:
For these, and the other functions that _were_ implemented and now removed, should we add them to the list of `Unsupported Operators` in the README?

Reply:
Good call!
I looked at the README and I don't really see a need to go into this level of details. I'll add a line stating that the driver does not support ANSI endpoints, that should be enough, no need to list all the ANSI functions.
But, I still need to also clean-up the list of unsupported functions.